src/app/ui/persmission-handler.js

```diff
@@ -41,6 +41,13 @@ export class PermissionHandler {
 
   constructor () {
     this.permissions = this._getFromLocal()
+    this.currentVersion = 1
+    // here we remove the old permissions saved before adding 'permissionVersion'
+    // since with v1 the structior has been changed because of new engine ^0.2.0-alpha.6 changes
+    if (!localStorage.getItem('permissionVersion')) {
```

Comment:
that still won't work like this, you have to do 
`if (localStorage.getItem('permissionVersion') !== this.currentVersion) {`

Reply:
if this.currentVersion is 2. or 4. the user loads and there is no 'permissionVersion' this would be undefined. ok i can leave the check for undefined. 
now lets say permissionVersion is 1 and we have currentVersion=2 now so 1 != 2 we gonna rm the old one(correct) and set it to 1(incorrect).

with my logic, if the 'permissionVersion' was not introduced it will rm the old permissions and set 1 to permissionVersion. 
if it is already there will not do anything